packages/server-wallet/src/__chain-test__/directly-funded-channel.test.ts

```diff
@@ -95,7 +95,6 @@ it('Create a directly funded channel between two wallets ', async () => {
   // PreFund0
   const preFundA = await a.createChannel(channelParams);
 
-  // TODO compute the channelId for a better test
```

Comment:
Is the test now a better test?

Reply:
I disagree with this necessarily being something we should do for a better test — we have other tests to ensure that channel IDs are being computed correctly inside the wallet, and an integration test does not seem the right place to check that implementation detail.